From: Andrea Zagli <azagli@libero.it>
Date: Mon, 4 Dec 2017 15:28:12 +0100
Subject: apache api version 2.2

---
 src/mod_authn_zakauthe.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/mod_authn_zakauthe.c b/src/mod_authn_zakauthe.c
index 2474b42..23837e1 100644
--- a/src/mod_authn_zakauthe.c
+++ b/src/mod_authn_zakauthe.c
@@ -129,9 +129,9 @@ static const authn_provider authn_zakauthe_provider =
 static void
 register_hooks (apr_pool_t *pool)
 {
-	ap_register_auth_provider (pool, AUTHN_PROVIDER_GROUP, "zakauthe",
-	                           AUTHN_PROVIDER_VERSION,
-	                           &authn_zakauthe_provider, AP_AUTH_INTERNAL_PER_CONF);
+	ap_register_provider (pool, AUTHN_PROVIDER_GROUP, "zakauthe",
+	                      "0",
+	                      &authn_zakauthe_provider);
 }
 
 /*
